packages/angular/package.json

```diff
@@ -33,37 +34,34 @@
   "license": "Apache-2.0",
   "author": "IBM",
   "peerDependencies": {
-    "@angular/common": "^9.0.0 || ^10.0.0 || ^11.0.0",
-    "@angular/core": "^9.0.0 || ^10.0.0 || ^11.0.0",
-    "@angular/forms": "^9.0.0 || ^10.0.0 || ^11.0.0",
+
     "carbon-components": "^10.0.0",
     "carbon-components-angular": "^4.0.0",
-    "rxjs": "^6.0.0",
-    "zone.js": "^0.8.26 || ^0.9.0 || ^0.10.0"
+    "rxjs": "^6.0.0"
   },
   "devDependencies": {
-    "@angular-builders/custom-webpack": "11.1.1",
-    "@angular-devkit/build-angular": "0.1101.1",
-    "@angular-devkit/build-ng-packagr": "0.1002.1",
-    "@angular-devkit/core": "11.1.1",
-    "@angular/animations": "11.1.1",
-    "@angular/cli": "11.1.1",
-    "@angular/common": "11.1.1",
-    "@angular/compiler": "11.1.1",
-    "@angular/compiler-cli": "11.1.1",
-    "@angular/core": "11.1.1",
-    "@angular/forms": "11.1.1",
-    "@angular/platform-browser": "11.1.1",
-    "@angular/platform-browser-dynamic": "11.1.1",
-    "@angular/router": "11.1.1",
+    "@angular-devkit/build-angular": "12.2.17",
+    "@angular-devkit/core": "12.2.17",
+    "@angular/animations": "12.2.16",
+    "@angular/cli": "12.2.17",
+    "@angular/common": "12.2.16",
+    "@angular/compiler": "12.2.16",
+    "@angular/compiler-cli": "12.2.16",
+    "@angular/core": "12.2.16",
+    "@angular/forms": "12.2.16",
+    "@angular/platform-browser": "12.2.16",
+    "@angular/platform-browser-dynamic": "12.2.16",
+    "@angular/router": "12.2.16",
     "@babel/core": "^7.13.16",
-    "@storybook/addon-actions": "^6.2.3",
-    "@storybook/addon-knobs": "^6.2.3",
-    "@storybook/addon-links": "^6.2.3",
-    "@storybook/addon-storysource": "^6.2.3",
-    "@storybook/addons": "^6.2.3",
-    "@storybook/angular": "^6.2.3",
-    "@storybook/theming": "^6.2.3",
+    "@storybook/addon-actions": "6.5.9",
+    "@storybook/addon-essentials": "6.5.9",
+    "@storybook/addon-links": "6.5.9",
+    "@storybook/angular": "6.5.9",
+    "@storybook/builder-webpack5": "6.5.9",
+    "@storybook/manager-webpack5": "6.5.9",
+    "@storybook/source-loader": "6.5.9",
+    "@storybook/addon-storysource": "6.5.9",
+    "@storybook/theming": "6.5.9",
```

Comment:
It's good that these are locked versions. 👍 
@Akshat55, should they be locked at 6.2.3 instead?